### ISSUE TITLE
[package] <glib> Generated pkg-conf file contains invalid content

### ISSUE THREAD

**Issue by planetmarshall** [conan-io/conan-center-index#12890]
### Package and Environment Details

* Package Name/Version: **glib/2.73.0**
* Conan version: **conan 1.52.0**

### Steps to reproduce

Create the following `conanfile.txt`

```
[requires]
glib/2.73.3

[generators]
PkgConfigDeps
```

Run
```
conan install .
```

The generated `gio-2.0/pc` file contains the following content

```
prefix=C:/.conan/85edc9/1
libdir1=${prefix}/lib
includedir1=${prefix}/include
# Custom PC content
datadir=${prefix}/res
schemasdir=${datadir}/glib-2.0/schemas
bindir=${prefix}/bin
giomoduledir=${libdir}/gio/modules
gio=${bindir}/gio
gio_querymodules=${bindir}/gio-querymodules
glib_compile_schemas=${bindir}/glib-compile-schemas
glib_compile_resources=${bindir}/glib-compile-resources
gdbus=${bindir}/gdbus
gdbus_codegen=${bindir}/gdbus-codegen
gresource=${bindir}/gresource
gsettings=${bindir}/gsettings

Name: gio-2.0
Description: Conan component: glib-gio-2.0
Version: 2.73.3
Libs: -L"${libdir1}" -lgio-2.0 -liphlpapi -ldnsapi -lshlwapi
Cflags: -I"${includedir1}"
Requires: glib-2.0 gobject-2.0 gmodule-2.0 zlib
```

The `libdir` variable - required by `giomoduledir` is undefined (`libdir1` is defined instead)

**Comment by SpaceIm** [comment]
Indeed, https://github.com/conan-io/conan-center-index/pull/12221 fixes this issue (but CI fails for other reasons).

**Comment by jwillikers** [comment]
@planetmarshall I believe this has been fixed in #13645.

**Comment by prince-chrismc** [comment]
Closed since the mentioned PR went it
